docs/TuxAnimations.md

```diff
@@ -0,0 +1,208 @@
+# Tux/Animations
+
+##Premise
+>Tux Animations is a library of default animation components and            easings. Additionally Tux Animations includes the makeAnimation function     which can be used to create custom animation components.
+
+
+##Implementation
+####1) Requiring Animations Components
+The Tux Animations interface including all animation defaults can be exposed via:
+
+    var Animations = require('tux/Animations');
+
+#####Require only what you need
+Tux is an extremely modular-conscious framework and as such it is possible to require only the minimal number of files needed in order to save space in your app. For Example, if you need access to the default Fly and FadeUp components in your implementation, you need only to expose those specific animations via:
```

Comment:
Change to `Tux is a modular framework`, change to `default Fly and FadeUp animations`, change to `you need only to require those specific animations via:`.